src/nb_ndarray.cpp

```diff
@@ -803,8 +804,12 @@ PyObject *ndarray_export(ndarray_handle *th, int framework,
     }
 
     if (copy) {
+        std::string copy_str("copy");
```

Comment:
This looks good, but can you please use a `const char *` here? There is no need for a `std::string`.

You can remove the braches (`{`) for the `if` statement, this codebase doesn't use them if the `then` condition is a simple one-liner.

Reply:
used const char*, removed braces, and squashed